specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2022-03-08-preview/Administrators.json

```diff
@@ -85,6 +85,9 @@
               "$ref": "#/definitions/ActiveDirectoryAdministrator"
             }
           },
+          "202": {
```

Comment:
>202 [](http://example.com/codeflow?start=11&length=3)

PUT MUST not return a 202

Reply:
In our RP we have a lot of PUT operations that return 202. We'll fix it later, but for now we need to keep 202 in this API, and we've had this issue before: https://github.com/Azure/azure-rest-api-specs/pull/21234#pullrequestreview-1159141156

Reply:
@rkmanda looped you in the chat history we had earlier with Tim during earlier review regarding PUT calls returning 202.